Author: Yongzhi Pan <panyongzhi@gmail.com>
Description: Fix resource directories
Last-Update: 2012-04-02
--- a/4digits
+++ b/4digits
@@ -44,8 +44,8 @@
 
 __version__ = '1.1'
 # We do not search glade file in /usr/share/ any longer. Debian packagers may need to patch this.
-__gladefile__ = os.path.join(os.path.dirname(__file__), '4digits.glade')
-__helpfile__ = os.path.join(os.path.dirname(__file__), 'doc', 'index.html')
+__gladefile__ = "/usr/share/4digits/4digits.glade"
+__helpfile__ = "/usr/share/doc/4digits/index.html"
 __appdata_dir__ = os.path.join(os.path.expanduser('~'), '.4digits')
 __config_path__ = os.path.join(__appdata_dir__, 'prefs.pickle')
 __score_filename__ = os.path.join(__appdata_dir__, '4digits.4digits.scores')
